### ISSUE TITLE
Divider line between Design tree and Code editor cell in workbooks / Devider line between Design tree and 3D view in inspector window is not visible  in High contrast modes

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#392]
### Steps to Reproduce

1. Change the High contrast standards before opening the application.
2. Launch Xamarin Workbooks application and create a new workbook.
3. Observe the Devider line in Workbooks and Inspector windows (Please refer the screenshots)

### Expected Behavior
Divider line should be visible
### Actual Behavior
Divider line between Tree tree and Code editor cell in workbooks / Divider line between Tree and 3D view is not visible in High contrast modes.
Note: In high contrast white mode Divider line gets dotted rectangular focus while navigating through the workbooks and inspector windows. But there is no indication of Divider line.
Please refer screenshots
### User Impact
User cannot see the Divider line. So user cannot resize the tree/editor/3d view areas
### Screenshots
![18 hcb inspector](https://user-images.githubusercontent.com/22956130/40774608-ac5a21a0-64e3-11e8-9e55-e8df0e576b78.png)
![18 hcb workbooks](https://user-images.githubusercontent.com/22956130/40774609-ac8e87b0-64e3-11e8-9081-76eab82b87b2.png)
![18 hcw inspector](https://user-images.githubusercontent.com/22956130/40774611-acb87e44-64e3-11e8-8d8e-e56efa688850.png)
![18 hcw workbooks](https://user-images.githubusercontent.com/22956130/40774612-aceb8b90-64e3-11e8-9b0c-a1c822756cf0.png)
![18 normal inspector](https://user-images.githubusercontent.com/22956130/40774613-ad1972b2-64e3-11e8-8320-35f1bf8c4c68.png)
![18 normal workbooks](https://user-images.githubusercontent.com/22956130/40774614-ad5164ba-64e3-11e8-9ede-8e914bdfa908.png)

![18 hcw dotted rectangle focus inspector](https://user-images.githubusercontent.com/22956130/40774751-0eacf260-64e4-11e8-848d-96d77212be71.png)
![18 hcw dotted rectangular focus workbooks](https://user-images.githubusercontent.com/22956130/40774752-0edaf6ba-64e4-11e8-9d5f-10b089cf7d5b.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
